.travis.yml

```diff
@@ -1,23 +1,18 @@
 language: node_js
 node_js:
-  - 8
-  - 10
-  - 12
+  - "8"
+  - "9"
```

Comment:
why is 9 being added ?

Reply:
with node only even numbers are considered stable

Reply:
I have reverted the changes but I am looking into why this could have happened..